Doc/source/designspaceLib/readme.rst

```diff
@@ -0,0 +1,953 @@
+DesignSpaceDocument

```

Comment:
This is `README.md` converted to reST using Pandoc. I'm sure that it could fit much better in Sphinx's model if it were re-organized by hand (maybe by moving most of it into docstrings?) but maybe it's for another PR?

Reply:
> moving most of it into docstrings?) but maybe it's for another PR?

yes, let's do this another time.